app/views/home/pii_phi_information.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="grid-container position-relative">
+  <div class="grid-row grid-gap">
+    <div class="desktop:grid-col-2 grid-col-auto z-bottom">&nbsp;</div>
+
+    <div id="instructions" class="desktop:grid-col-10 grid-col-12 p0">
+      <section class="margin-bottom-3">
+        <%= render "shared/messages" %>
+        <h1 class="font-sans-2xl line-height-sans-205 margin-top-5">Personally Identifiable Information and Protected
+          Health Information</h1>
+      </section>
+      <section class="margin-bottom-3">
+        <h2 id="pii_top" class="margin-0 margin-bottom-2 text-bold font-sans-xl">What is Personally Identifiable
+          Information (PII)?</h2>
+        <div class="font-sans-sm desktop:grid-col-8">
+          <p>PII includes the following:</p>
+          <ul class="line-height-26 font-sans-sm padding-top-0 padding-left-205 margin-0">
+            <li>First and last name</li>
+            <li>Email address</li>
+            <li>Phone number</li>
+            <li>Photographs</li>
+            <li>Social security number</li>
+            <li>Maiden name</li>
+            <li>Date of birth</li>
+            <li>Combination of home address and name or any other identifiable information to someone's name</li>
+          </ul>
+        </div>
+      </section>
+      <section class="margin-bottom-3">
+        <h2 class="margin-0 margin-bottom-2 text-bold font-sans-xl">What is Protective Health Information (PHI)</h2>
+        <div class="font-sans-sm desktop:grid-col-8">
+          <p>PHI includes the following:</p>
+          <ul class="line-height-26 font-sans-sm padding-top-0 padding-left-205 margin-0">
+            <li>Medical information</li>
+            <li>Diagnosis</li>
+          </ul>
+        </div>
+      </section>
+      <section class="margin-bottom-3">
+        <h2 class="margin-0 margin-bottom-2 text-bold font-sans-xl">Consent forms</h2>
+        <ul class="line-height-26 font-sans-sm padding-top-0 padding-left-205 margin-0 desktop:grid-col-8">
+          <li>Do not enter Personally Identifiable Information (PII) or Protected Health Information (PHI) for any
```

Comment:
the copy from figma is different than what's here just wanted to flag it but am assuming this is the correct one.

Reply:
yeah, same comment as above :smile: